protected/humhub/modules/content/components/ContentContainerModule.php

```diff
@@ -219,4 +219,15 @@ protected function getGlobalPermissions()
         return [];
     }
 
+    /**
+     * This method is called to determine classes of Content models which can be posted on wall.
+     *
+     * @param ContentContainerActiveRecord|null $contentContainer
+     * @return array
+     */
+    public function getContentClasses(?ContentContainerActiveRecord $contentContainer = null): array
+    {
+        return [];
```

Comment:
Consideration: Would it perhaps be better to return an empty e.g. `Post` object here instead of just the class name as string. 

This way one could also cover cases with "dynamic" content objects.
 
Besides, we then have to instantiate anyway to get the name/canEdit/icon/etc.

Reply:
@luke- Do you mean the code like? :

```php
return [new Post($contentContainer)]
```